lambdas/issue-client-access-token/src/main/java/uk/gov/di/ipv/core/issueclientaccesstoken/validation/TokenRequestValidator.java

```diff
@@ -61,10 +61,10 @@ public void authenticateClient(String requestBody) throws ClientAuthenticationEx
     private void validateMaxAllowedAuthClientTtl(JWTAuthenticationClaimsSet claimsSet)
             throws InvalidClientException {
         Date expirationTime = claimsSet.getExpirationTime();
-        String maxAllowedTtl = configService.getSsmParameter(MAX_ALLOWED_AUTH_CLIENT_TTL);
+        long maxAllowedTtlSeconds =
+                Long.parseLong(configService.getSsmParameter(MAX_ALLOWED_AUTH_CLIENT_TTL));
 
-        OffsetDateTime offsetDateTime =
-                OffsetDateTime.now().plusSeconds(Long.parseLong(maxAllowedTtl));
+        OffsetDateTime offsetDateTime = OffsetDateTime.now().plusSeconds(maxAllowedTtlSeconds);
```

Comment:
Was there a specific reason for this refactor out of interest? Or just readability?

Reply:
Just readability :)